app/models/user_project_role.rb

```diff
@@ -0,0 +1,15 @@
+class UserProjectRole < ActiveRecord::Base
```

Comment:
To sum up, following all the above considerations, I'll keep it as it is, if you're ok with that @henders.

But I'm still open to discussion so let me know if, in the meantime, you looked into this and have any further thoughts.